tljh/jupyterhub_config.py

```diff
@@ -49,3 +50,9 @@ def start(self):
 else:
     config_overrides = {}
 configurer.apply_config(config_overrides, c)
+
+# Load arbitrary .py config files if they exist.
+# This is our escape hatch
+extra_configs = sorted(glob(os.path.join(INSTALL_PREFIX, 'config.d', '*.py')))
```

Comment:
Let's call it 'jupyterhub_config.d' to make it clear what it configures

Reply:
That makes sense, @minrk! Will do.

Reply:
Oh - does `.d` indicate for historial reasons that a file is a partial file?

Reply:
@consideRatio yep! I picked it up from linux distros like debian - you have a /etc/sudoers file and a /etc/sudoers.d directory that can contain partials.

Reply:
Ah! I love when a piece like that falls into place! :heart: @minrk @yuvipanda

Reply:
Done, @minrk